### ISSUE TITLE
Can only use /home (and /media?)

### ISSUE THREAD

**Issue by panlinux** [popey/gphotos-sync-snap#1]
Hi,

this might not be solvable in the snap, because, snaps. But I find it very restrictive to only be able to use `/home` to store the downloaded media.

I have a backup server where I wanted to store my google photos data, and I'm obviously not mounting its directories under `/home`, but elsewhere (`/storage` for that matter). This snap cannot use that mountpoint.

**Comment by popey** [comment]
One alternative is to use a bind mount. 

```
sudo mkdir /media/photos
sudo mount -o bind /storage/photos /media/photos
```

Using the `removable-media` interface, the application will be able to see the files under `/media/photos`.

**Comment by panlinux** [comment]
I understand there are workarounds, I just wished there was something more "native" to the snap, like me, the user who installed it (as root), telling it that it's ok to use a particular directory on my system. In fact, I would *not* want it to be able to read my `/home`.

**Comment by popey** [comment]
Yeah, sorry about that. Perhaps start a thread with a strawman proposal for how you would expect to see this, over on the snapcraft forum?
